### DISCUSSION TITLE
Mongodb+Prisma 3.7.0:: many-many am i doing it wrong?

### DISCUSSION THREAD

**Discussion by renenielsendk** [prisma/prisma#10954] [Q&A] [answered]
Hi all.

So the thing is i have a platform that has courses, one course can have many topics.
Same way as a topic can have many courses.

I currently have a schema that looks like this: https://gist.github.com/Renetnielsen/f040711e6187a1f4f1dd035c8ab615ea

When i try to find all courses for one topic, i always only the courses which is attached to one topic and not several.
I created a seed script to illustrate, and the output.

Please help me in what am i doing wrong?.

You should be able to reproduce using the 3 scripts below.

**script** https://gist.github.com/Renetnielsen/8c342d3cb6c335bb535b5c8a4ce2c426
**output** https://gist.github.com/Renetnielsen/29a2baedfe7e461742f5b8774ab61108
**schema** https://gist.github.com/Renetnielsen/f040711e6187a1f4f1dd035c8ab615ea

**Comment by renenielsendk** [comment, marked as answer]
using `hasEvery` solved the issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
